biolink-model.yaml

```diff
@@ -37,11 +42,18 @@ prefixes:
   ICD10: 'http://translator.ncats.nih.gov/ICD10_'
   ICD9: 'http://translator.ncats.nih.gov/ICD9_'
   medgen: 'https://www.ncbi.nlm.nih.gov/medgen/'
+  ISBN: 'https://isbn-international.org/'
+  ISSN: 'https://portal.issn.org/resource/ISSN/'
+  MESH_PUB: 'https://www.nlm.nih.gov/mesh/pubtypes.html#'
```

Comment:
this is part of mesh proper?

Reply:
Hmm... so it is!. I'll revert to using the bare MESH prefix.